docs/3.0rc/develop/write-flows.mdx

```diff
@@ -65,11 +98,204 @@ Forks 🍴 : 1245
+
+```python
+from prefect import flow, task
+
+@task
+def print_hello(name):
+    print(f"Hello {name}!")
+
+@flow(name="Hello Flow")
+def hello_world(name="world"):
+    print_hello(name)
+```
+
+A single flow function can contain all of your workflow's code.
+However, if you put all of your workflow logic in a single flow function and any line of code fails, the entire flow fails and must be retried from the beginning. 
+Organizing your workflow code into smaller flow and task units lets you take advantage of Prefect features like retries, 
+more granular visibility into runtime state, the ability to determine final state regardless of individual task state, and more.
+
+You may call any number of tasks, other flows, and even regular Python functions within your flow. 
+You can pass parameters to your flow function to use elsewhere in the workflow, and Prefect will report on the progress 
+and [final state](#final-state-determination) of any invocation.
+
+We reccomend "small tasks." 
+Each task should be a single discrete piece of work in your workflow. 
+For example, calling an API, performing a database operation, analyzing or transforming a data point. 
+The more granular you make your tasks, your workflows can better recover from failures and you can more easily find and fix issues.
+Prefect tasks are well suited for parallel or distributed execution using distributed computation frameworks such as Dask or Ray. 
+
+### Nesting flows
+
+In addition to calling tasks within a flow, you can also call other flows.
+A _nested_ flow run is created when a flow function is called inside the execution of another flow.
+When one flow calls another, the calling flow run is the "parent" run, the called flow run is the "child" run.
+
+Nesting flows is a great way to organize your workflows and offer more visibility within the UI.
+In the UI, each child flow run is linked to its parent and can be individually observed.
+
+For most purposes, nested flow runs behave just as all flow runs do.
+There is a full representation of the nested flow run in the backend as if it had been called separately. 
+Nested flow runs differ from normal flow runs in that they resolve any passed task futures into data.
+This allows data to be passed from the parent flow run to the child run easily.
+
+When a nested flow run starts, it creates a new [task runner](/3.0rc/develop/task-runners/) for any tasks it contains.
+When the nested flow run completes, the task runner shuts down. 
+Nested flow runs block execution of the parent flow run until completion.
+However, asynchronous nested flows can run concurrently with [AnyIO task groups](https://anyio.readthedocs.io/en/stable/tasks.html) or 
+[asyncio.gather](https://docs.python.org/3/library/asyncio-task.html#id6).
+
+The relationship between a child and parent flow run is tracked by creating a special task run in the parent flow run.
```

Comment:
This seems confusing, maybe worth calling out more. A Flow run creates a task run to track its state. Maybe adding a schematic as a future to do?

Reply:
I think we should just remove this entirely. Its only relevant to the flow run graph and its obvious when you look at it.